### PR TITLE
feat: add commands for SCF Alias

### PR DESCRIPTION
**Pull request by Jominja**

### CODE REVIEW

File: serverless.component.yml [hunk 1]
```diff
@@ -235,3 +235,15 @@ actions:
                     type: string
   remove:
     definition: Remove your Express.js application
+  list_alias:
+    definition: Show your aliases
```
Comment (timqian) [inline serverless.component.yml:239]: If I remember it right, in yesterday's product meeting, we have decided to use Chinese for all the descriptions in serverless.component.yml and have a plan to auto-generate configuration.md according to this file.

cc @tinafangkunding @AprilJC @yugasun

Comment (Jominja) [reply serverless.component.yml:239]: If use Chinese to describe the COMMANDs,i need the  descriptions from @tinafangkunding

**Comment by yugasun** [comment]
@Jominja You should update version of [tencent-component-toolkit](https://github.com/serverless-tencent/tencent-component-toolkit) to latest version in [package.json](https://github.com/serverless-components/tencent-scf/blob/master/src/package.json).

File: src/serverless.js [hunk 2]
```diff
@@ -145,6 +145,192 @@ class ServerlessComponent extends Component {
...
+      const region = inputs.region || CONFIGS.region
+
+      const publish_params = {}
+      publish_params.functionName = inputs.function
+      publish_params.namespace = inputs.namespace
+      publish_params.description = inputs.description
+
+      const scf = new Scf(credentials, region)
+
+      const scfOutput = await scf.publishVersion(publish_params)
```
Comment (yugasun) [inline src/serverless.js:293]: Publish version of SCF is an asynchronous flow, maybe you should wait for the `Active` status of SCF.

File: src/serverless.js [hunk 3]
```diff
@@ -145,6 +145,194 @@ class ServerlessComponent extends Component {
...
+        }
+        alias_unit[aliases[i].Name].weight = JSON.stringify(weight)
+
+        listAlias.alias.push(alias_unit)
+      }
+      console.log(listAlias)
+      return listAlias
+    } catch (e) {
+      return {
+        requestId: e.reqId,
```
Comment (yugasun) [inline src/serverless.js:192]: All errors except api error in components code should throw `TypeError` like:

```
throw new TypeError(
        'xxx',
        `xxx`
 )
```

If you do not throw error, users will get success status even though they get the error message for cli.

**Pull request merged**